tools/report-converter/codechecker_report_converter/sparse/output_parser.py

```diff
@@ -0,0 +1,91 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+
+import logging
+import os
+import re
+
+from ..output_parser import Message, Event, BaseParser
+
+LOG = logging.getLogger('ReportConverter')
+
+
+class SparserParser(BaseParser):
```

Comment:
Minor comment/question: is this `SparserParser` instead of `SparseParser` intentionally?